### ISSUE TITLE
Pengwin can not launch after tranform to WSL2

### ISSUE THREAD

**Issue by SukkaW** [WhitewaterFoundry/Pengwin#499]
**Note**

These WSL error messages occur from time to time and can tricky to troubleshoot short of reinstalling WSL.

They are usually not an issue with Pengwin itself but a glitch with the underlying Windows Subsystem for Linux in Windows 10.

You can help us help you and Microsoft improve WSL and address these if you can provide us some debugging information.

- [x] I have read the above statement.

**Basic Troubleshooting**

Please check:

- [x] I have confirmed WSL is enabled: https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#confirm-wsl-is-enabled.
- [ ] I have searched Google for the error message.

> Actually there are no error msg at all.

- [x] I have searched the official Microsoft WSL issues page: 

I have used `WSL2` as a keyword but found nothing useful.

- [x] I have searched the Pengwin issues page:

> I have used `WSL2` as a keyword but found nothing useful either.

- [x] I have reset Pengwin: Settings->Apps->Apps & features->Pengwin->Advanced Options->Reset

Everytime I reset, the Pengwin become missing in `Settings > Application`. I have to uninstall and reinstall Pengwin again.

- [ ] I have disabled and re-enabled WSL in Windows Features.
- [x] I have run Windows 10 updates and restarted.

**Version**

Find: Settings->Apps->Apps & features->Pengwin->Advanced Options->Version.

Insert here: `1.2.7.0`

**Screenshot**

Insert here:

**Windows Build**

Run 'systeminfo | findstr /C:"OS"' in Command Prompt and insert here:

```
OS Name:                   Microsoft Windows 10 Enterprise Insider Preview
OS Version:                10.0.18945 N/A Build 18945
OS Manufacturer:       Microsoft Corporation
OS Configuration:      Standalone Workstation
OS Build Type:            Multiprocessor Free
BIOS Version:              LENOVO R0PET58W (1.35 ), 2019/7/2
```

---

After I setup my unix username & password in Pengwin, I transfered Pengwin from WSL 1 to WSL 2 using the command below in Powershell:

```
wsl --set-version WLinux 2
```

After the transformation is finished, the Pengwin can not launch any more. Everytime I open Pengwin, the `Loading Pengwin` window show up, and then disappeared shortly without any error code.

**Comment by SukkaW** [comment]
After I transform Pengwin back to WSL 1:

```
wsl --set-version WLinux 1
```

Then Pengwin can launch.

**Comment by patrick330602** [comment]
This is a known WSL 2 Bug in the latest Windows Insider build, for more information see issue #497

**Comment by hMounir** [comment]
I have the same exact issue, but I found out that the internet is not working, try to click on Ctrl + C and you will see you will get the prompt command. I don't know how to solve it

**Comment by sirredbeard** [comment]
Closed as duplicate of #497.
